manifest.json

```diff
@@ -17,6 +17,7 @@
        "128": "icons/icon_128.png" 
     },
     "permissions": [
+        "*://linkedin.com/*",
```

Comment:
Is there any way to NOT have this permission in the manifest? I'd love to see some guides where you learned this! I'm a little skeptical about it having any permissions more than activeTab because chrome webstore starts warning users that the extension might be able to read browsing history and such but if this the only way, I'm fine with it!

Reply:
Hi! It should be fine to remove. I think I left it in from another approach I was trying. I'll do it now!